superset-frontend/packages/superset-ui-core/test/query/buildQueryContext.test.ts

```diff
@@ -18,6 +18,7 @@
  */
 import { buildQueryContext } from '@superset-ui/core';
 import * as queryModule from '../../src/query/normalizeTimeColumn';
+import * as getXAxisModule from '../../src/query/getXAxis';
```

Comment:
I know this is following the same convention as on the line above, but I personally prefer to avoid `import *` whenever possible.

Reply:
The module is used to mock an object, so we have to do `import * as moduleName from 'reletive path'` statement here.

Reply:
Oh yes, makes sense 👍